Dockerfile.import

```diff
@@ -0,0 +1,12 @@
+FROM debian:stretch
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
```

Comment:
We need to do the debian update now twice, once for the import step and once when we run. Doing the debian update is one of the slowest steps on my connection. Is there a way we can do the update only once?